cpp/splinepy/py/init/reader.cpp

```diff
@@ -2,6 +2,6 @@
 
 namespace splinepy::py::init {
 
-void init_reader(py::module_& m) { add_spline_reader(m); }
+void init_reader(py::module_& m) { AddSplineReader(m); }
```

Comment:
This should probably be a separaate PR. can you set it back to what it was before?

Reply:
Also changed the function name in `cpp/splinepy/py/py_spline_reader.hpp` to `add_spline_reader`, as it was before.